wazo_confd/plugins/extension/validator.py

```diff
@@ -185,6 +185,70 @@ def validate(self, extension):
                 associated_id=typeval,
             )
 
+        if extension.conference:
+            raise errors.resource_associated(
+                'Extension',
+                'conference',
+                extension_id=extension.id,
+                associated_id=extension.conference.id,
+            )
+
+        if extension.parking_lot:
+            raise errors.resource_associated(
+                'Extension',
+                'parking_lot',
+                extension_id=extension.id,
+                associated_id=extension.parking_lot.id,
+            )
+
+        if extension.group:
+            raise errors.resource_associated(
+                'Extension',
+                'group',
+                extension_id=extension.id,
+                associated_id=extension.group.id,
+            )
+
+        if extension.lines:
+            if len(extension.lines) == 1:
+                raise errors.resource_associated(
+                    'Extension',
+                    'line',
+                    extension_id=extension.id,
+                    associated_id=extension.lines[0].id,
+                )
+            else:
+                raise errors.resource_associated(
+                    'Extension',
+                    'lines',
+                    extension_id=extension.id,
+                    associated_ids=[line.id for line in extension.lines],
+                )
```

Comment:
Is there a reason why the distinction between one line and multiple is important?

Reply:
Fixed, as there is no specific reason